nativelib/src/main/scala/scala/re2s/Compiler.scala

```diff
@@ -0,0 +1,270 @@
+// Copyright 2010 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Original Go source here:
+// http://code.google.com/p/go/source/browse/src/pkg/regexp/syntax/compile.go
+
+package scala.re2s
```

Comment:
`scala.scalanative.re2s` is a better package name here. We can't jus tadd things to `scala` namespace.

Reply:
Changed as suggested. Builds in Travis. 
Thank you.  By the light of day, "What was I thinking?"

This is _exactly_ the kind of thing I was trying to detect in this PR.